File: tests/fixtures/python/composition_aggregation.py
class Engine:
    def __init__(self, horsepower: int) -> None:
        self.horsepower: int = horsepower


class Wheel:
    def __init__(self, size: int) -> None:
        self.size: int = size


class Person:
    def __init__(self, name: str) -> None:
        self.name: str = name


class Car:
    def __init__(self, owner: Person) -> None:
        self.engine = Engine(300)
        self.owner = owner
        self.wheels: list[Wheel] = []

    def register_owner(self, owner: Person) -> None:
        pass

File: tests/fixtures/java/composition_aggregation.java
public class Engine {
    private int horsepower;

    public Engine(int horsepower) {
        this.horsepower = horsepower;
    }
}

public class Wheel {
    private int size;

    public Wheel(int size) {
        this.size = size;
    }
}

public class Person {
    private String name;

    public Person(String name) {
        this.name = name;
    }
}

public class Car {
    private Engine engine;
    private Person owner;
    private List<Wheel> wheels;

    public Car(Person owner) {
        this.engine = new Engine(300);
        this.owner = owner;
    }

    public void registerOwner(Person owner) {
    }
}
